src/Features/CSharp/Portable/TaskList/CSharpTaskListService.cs

```diff
@@ -11,29 +11,24 @@
 using Microsoft.CodeAnalysis.Host;
 using Microsoft.CodeAnalysis.Host.Mef;
 using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.CodeAnalysis.TaskList;
 using Microsoft.CodeAnalysis.TodoComments;
 using Roslyn.Utilities;
 
-namespace Microsoft.CodeAnalysis.CSharp.TodoComments
+namespace Microsoft.CodeAnalysis.CSharp.TaskList
 {
-    [ExportLanguageServiceFactory(typeof(ITodoCommentService), LanguageNames.CSharp), Shared]
-    internal class CSharpTodoCommentServiceFactory : ILanguageServiceFactory
+    [ExportLanguageService(typeof(ITodoCommentService), LanguageNames.CSharp), Shared]
+    internal class CSharpTaskListService : AbstractTaskListService
```

Comment:
no need for factory.  made into only a normal service export. (same will happen for vb).